src/hotspot/cpu/riscv/riscv_v.ad

```diff
@@ -123,6 +120,69 @@ instruct storeV(vReg src, vmemA mem) %{
   ins_pipe(pipe_slow);
 %}
 
+// vector load mask
+
+instruct vloadmask(vRegMask dst, vReg src) %{
+  match(Set dst (VectorLoadMask src));
+  format %{ "vloadmask $dst, $src" %}
+  ins_encode %{
+    __ vsetvli(t0, x0, Assembler::e8);
+    __ vmsne_vx(as_VectorRegister($dst$$reg), as_VectorRegister($src$$reg), x0);
+  %}
+  ins_pipe(pipe_slow);
+%}
+
+instruct vloadmask_masked(vRegMask dst, vReg src, vRegMask_V0 vmask) %{
+  match(Set dst (VectorLoadMask src vmask));
+  format %{ "vloadmask_masked $dst, $src, $vmask" %}
+  ins_encode %{
+    __ vsetvli(t0, x0, Assembler::e8);
+    __ vmsne_vx(as_VectorRegister($dst$$reg), as_VectorRegister($src$$reg), x0, Assembler::v0_t);
+  %}
+  ins_pipe(pipe_slow);
+%}
+
+// vector store mask
+
+instruct vstoremask(vReg dst, vRegMask_V0 src, immI size) %{
+  match(Set dst (VectorStoreMask src size));
+  format %{ "vstoremask $dst, $src" %}
+  ins_encode %{
+    __ vsetvli(t0, x0, Assembler::e8);
+    __ vmv_v_x(as_VectorRegister($dst$$reg), x0);
+    __ vmerge_vim(as_VectorRegister($dst$$reg), as_VectorRegister($dst$$reg), 1);
+  %}
+  ins_pipe(pipe_slow);
+%}
+
+// vector mask compare
+
+instruct vmaskcmp(vRegMask dst, vReg src1, vReg src2, immI cond) %{
+  match(Set dst (VectorMaskCmp (Binary src1 src2) cond));
+  format %{ "vmaskcmp_rvv $dst, $src1, $src2, $cond" %}
```

Comment:
Suggestion: s/vmaskcmp_rvv/vmaskcmp/

Reply:
Thanks for the review! We will fix it in the next commit.